src/services/userService.ts

```diff
@@ -64,13 +64,57 @@ export default class UserService {
         return this.docRefsObservableToUsersObservable(this.db.updateDocs(USER_COLLECTION, users));
     }
 
-    approveUser(id: string): Observable<User> {
+    updateUserApproval(id: string, willBeApproved: boolean): Observable<User> {
```

Comment:
so almost there with what I was suggesting last review. We could cut the amount of code we write by a third if we do something like this
```
updateUserAproval(id: string, isApproved: boolean): Observable<User> {
    const updates = ids.map(id => ({
            id,
            isApproved
    });
    return this.docRefsObservableToUsersObservable(this.db.updateDocs(USER_COLLECTION, updates));
}